File: src/Sekiban.Web/Common/SekibanAggregateInfo.cs
namespace Sekiban.Web.Common;

/// <summary>
///     Aggregate information
/// </summary>
/// <param name="AggregateName"></param>
/// <param name="QueryInfo"></param>
/// <param name="Commands"></param>
public record SekibanAggregateInfo(string AggregateName, SekibanQueryInfo QueryInfo, List<SekibanCommandInfo> Commands);

File: internalUsages/customer-front/src/openapi/models/SekibanAggregateInfo.ts
/* istanbul ignore file */
/* tslint:disable */
/* eslint-disable */

import type { SekibanCommandInfo } from './SekibanCommandInfo';
import type { SekibanQueryInfo } from './SekibanQueryInfo';

export type SekibanAggregateInfo = {
    aggregateName?: string | null;
    queryInfo?: SekibanQueryInfo;
    commands?: Array<SekibanCommandInfo> | null;
};

